src/apps/Settings/Settings.js

```diff
@@ -181,9 +182,11 @@ class Settings extends React.Component {
                 <Text size="small" style={{ marginLeft: '10px' }}>
                   {(() =>
                     network.type === 'unknown'
-                      ? 'You are using an unsupported network. Please switch to rinkeby or a local test environment instead.'
+                      ? 'This app was built to connect to an unsupported network. Please change the network environment settings.'
```

Comment:
I realized the previous message didn't really make sense; this would only happen if the app was expecting to connect to an unknown network (which would be a environment configuration issue).

Reply:
Maybe we should replace “build” by “configured”? At first I understood that the app was built (designed) to connect to an unsupported network.